cachito/web/api_v1.py

```diff
@@ -342,12 +348,19 @@ def create_request():
     cachito_metrics["gauge_state"].labels(state="total").inc()
     cachito_metrics["gauge_state"].labels(state=request.state.state_name).inc()
 
+    ctx = trace.get_current_span().get_span_context()
+    trace_id = '{trace:032x}'.format(trace=ctx.trace_id)
```

Comment:
For my understanding: trace IDs are (up to) 128-bit numbers, so we use 32 hexadecimals to represent them? With 0-padding to make sure the string is always exactly 32 characters?

(could you add a comment to explain it in the code?)